tr/about/website/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: "Ruby Websitesi Hakkında"
+lang: tr
+---
+
+Bu websitesi, [Jekyll][jekyll] kullanılarak Ruby ile oluşturulmuştur,<br>
+kaynağı [GitHub][github-repo]'da barındırılmaktadır.
+
+Görsel tasarım [Jason Zimdars][jzimdars] tarafından yapılmıştır.<br>
+Ruby Visual Identity Team'in daha önceki bir tasarımı tabanlıdır.
+
+[Ruby logosu][logo] Telif Hakkı &copy; 2006, Yukihiro Matsumoto.
+
+
+## Sorunları Bildirme ##
+
+Bir sorunu bildirmek için [konu takibi][github-issues]ni kullanın ya da [webmaster][webmaster] ile irtibata geçin (İngilizce olarak).
```

Comment:
Sorun bildirmek için daha güzel bence ya da herhangi bir sorun bildirmek için.

Reply:
b661bfc